src/um/processsnapshot.rs

```diff
@@ -51,6 +53,40 @@ ENUM!{enum PSS_DUPLICATE_FLAGS {
 }}
 DECLARE_HANDLE!(HPSS, HPSS__);
 DECLARE_HANDLE!(HPSSWALK, HPSSWALK__);
+
```

Comment:
There shouldn't be any blank lines in the Rust source files. This patch adds 3 empty lines.